vignettes/wfs_wcs.Rmd

```diff
@@ -84,7 +91,7 @@ Here, we could have also used `sf::read_sf()` in combination with an [OGR SQL qu
 
 ```{r}
 prov2 <- read_sf(paste0("WFS:", wfs_vrbg),
-  query = "select * from Refprv where NAAM='West-Vlaanderen'"
+  query = "SELECT * FROM Refprv WHERE NAAM='West-Vlaanderen'"
```

Comment:
Conventionally, SQL statements capitalize all command components for readability. I am quite used to that, and adjusted herein, guessing that CQL would benefit from the same convention.
Feel free to disagree!